cypress-io/cypress
# chore: added support for --runner-ui and --no-runner-ui cli options

### Additional details

* Added new run mode cli only options, `--runner-ui` and `--no-runner-ui`
* Added new `runnerUi` parameter to the module API
* In addition to hiding the command log, when the runner is hidden, the runner header is also hidden.
* If the protocol is enabled, the runner is hidden:
  * passing `--runner-ui` will override this behavior and show the runner

### Steps to test

* In run mode, pass `--no-runner-ui` and verify the runner is not displayed
* With protocol enabled, pass `--runner-ui` and verify the runner is displayed

### How has the user experience changed?

Protocol enabled (runner is hidden by default):
<img width="1571" alt="Screenshot 2023-08-17 at 11 42 18 AM" src="https://github.com/cypress-io/cypress/assets/2002044/a629acae-7abb-408c-8e5a-8c644cc656da">

Protocol enabled passing `--runner-ui`
<img width="1986" alt="Screenshot 2023-08-17 at 11 43 15 AM" src="https://github.com/cypress-io/cypress/assets/2002044/89570c82-3c0b-47db-8c5d-eba0ddc321d9">

### PR Tasks
